pkg/services/searchV2/bluge.go

```diff
@@ -293,11 +293,15 @@ func getDashboardPanelIDs(reader *bluge.Reader, dashboardUID string) ([]string,
 	return panelIDs, err
 }
 
-func getFolderDashboardIDs(reader *bluge.Reader, folderUID string) ([]string, error) {
-	var dashboardIDs []string
+func getDocsIDsByLocationPrefix(reader *bluge.Reader, kinds []entityKind, prefix string) ([]string, error) {
```

Comment:
for delete... why do we care about multiple kinds?

Isn't it always everything with that prefix?